pom.xml

```diff
@@ -2,393 +2,419 @@
-        <pass.fedora.baseurl>http://${docker.host.address}:${fcrepo.http.port}/fcrepo/rest/</pass.fedora.baseurl>
-        <fcrepo.jms.baseurl>http://fcrepo:${fcrepo.http.port}/fcrepo/rest/</fcrepo.jms.baseurl>
-        <!--suppress UnresolvedMavenProperty -->
-        <pass.elasticsearch.host>${docker.host.address}</pass.elasticsearch.host>
-        <pass.elasticsearch.url>http://${pass.elasticsearch.host}:${es.port}/pass/</pass.elasticsearch.url>
-        <pass.elasticsearch.limit>100</pass.elasticsearch.limit>
-        <pass.doi.service.port>8080</pass.doi.service.port>
-    </properties>
+    <!-- other integration test related properties -->
+    <pass.jsonld.context>https://oa-pass.github.io/pass-data-model/src/main/resources/context-3.4.jsonld
+    </pass.jsonld.context>
+    <fcrepo.skip>false</fcrepo.skip>
+    <indexer.skip>false</indexer.skip>
+    <!--suppress UnresolvedMavenProperty -->
+    <fcrepo.server>${docker.host.address}</fcrepo.server>
+    <pass.fedora.user>fedoraAdmin</pass.fedora.user>
+    <pass.fedora.password>moo</pass.fedora.password>
+    <!--suppress UnresolvedMavenProperty -->
+    <pass.fedora.baseurl>http://${docker.host.address}:${fcrepo.http.port}/fcrepo/rest/</pass.fedora.baseurl>
+    <fcrepo.jms.baseurl>http://fcrepo:${fcrepo.http.port}/fcrepo/rest/</fcrepo.jms.baseurl>
+    <!--suppress UnresolvedMavenProperty -->
+    <pass.elasticsearch.host>${docker.host.address}</pass.elasticsearch.host>
+    <pass.elasticsearch.url>http://${pass.elasticsearch.host}:${es.port}/pass/</pass.elasticsearch.url>
+    <pass.elasticsearch.limit>100</pass.elasticsearch.limit>
+    <pass.doi.service.port>8080</pass.doi.service.port>
 
-    <profiles>
-        <profile>
-            <id>external-fcrepo-server</id>
-            <activation>
-                <property>
-                    <name>fcrepo.server</name>
-                </property>
-            </activation>
-            <properties>
-                <fcrepo.skip>true</fcrepo.skip>
-                <pass.fedora.baseurl>http://${fcrepo.server}:${fcrepo.http.port}/fcrepo/rest/</pass.fedora.baseurl>
-            </properties>
-        </profile>
-        <profile>
-            <id>standard</id>
-            <activation>
-                <property>
-                    <name>standard</name>
-                </property>
-            </activation>
-            <properties>
-                <fcrepo.http.port>8080</fcrepo.http.port>
-                <fcrepo.jms.port>61616</fcrepo.jms.port>
-                <es.port>9200</es.port>
-            </properties>
-        </profile>
-    </profiles>
+    <failOnMissingWebXml>false</failOnMissingWebXml>
+  </properties>
 
-    <build>
-        <finalName>pass-doi-service</finalName>
-        <plugins>
-            <plugin>
-                <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-maven-plugin</artifactId>
-            </plugin>
+  <dependencyManagement>
```

Comment:
I'd suggest adding a comment here to call out the purpose of this section (as an alternative to using the Spring boot parent pom). Otherwise, it'll be easy to forget why this change was made.